src/checkstyle/checkstyle.xml

```diff
@@ -150,16 +153,14 @@ page at http://checkstyle.sourceforge.net/config.html -->
         <module name="JavadocMethod">
```

Comment:
In `JavadocMethod`, the following parameters are no longer valid:
`allowMissingJavadoc`, `allowMissingThrowsTags`, `allowThrowsTagsForSubclasses` and `allowUndeclaredRTE`.
See https://github.com/checkstyle/checkstyle/issues/7329 for details.